### PR TITLE
Update footer links

### LINKED ISSUE #64: Update the Accessibility statement
### Description of the issue

When running both parent and plugin, the accessibility statement page is using the default datagateway values.

These are defined in: https://github.com/ral-facilities/scigateway/blob/develop/public/res/default.json

### LINKED ISSUE #65: Some footer links are dead
### Description of the issue

Some footer links will be dead as they default to `/#` These are defined in https://github.com/ral-facilities/scigateway/blob/develop/public/res/default.json

This is when both parent and plugin are deployed.

### PR DESCRIPTION
**Pull request by Dagonite**
Closes #65, closes #64.

## How to test

1. Replace SciGateway's `public/res` folder with the contents of this repo's `container` folder.
2. Run the web app.
3. See if clicking the footer links does anything. Also check if the accessibility statement page reads **correct**.

### CODE REVIEW

File: container/default.json [hunk 1]
```diff
@@ -128,21 +128,21 @@
...
     "alternative-text": "Science and Technology Facilities Council"
   },
   "accessibility-page": {
-    "title": "Accessibility statement for DataGateway",
-    "domains-list-text": "This accessibility statement applies to all facilities using DataGateway. This includes:",
-    "domains-list": ["https://data.isis.stfc.ac.uk "],
-    "website-developed-by": "These sites are maintained by the Data Software and Engineering Group, in the Scientific Computing Department. ",
+    "title": "Accessibility statement for FIA",
+    "domains-list-text": "This accessibility statement applies to all facilities using FIA. This includes:",
+    "domains-list": ["fia"],
```
Comment (Pasarus) [inline container/default.json:144]:
```suggestion
    "domains-list": ["reduce.isis.cclrc.ac.uk"],
```
Probably want to use this here

File: container/default.json [hunk 2]
```diff
@@ -128,21 +128,21 @@
   "footer": {
     "links": {
       "text": "<0>ISIS Home</0> | <2>Privacy statement</2> | <4>Data policy</4> | <6>Accessibility statement</6> | <8>Contact</8>",
-      "facility": "#",
-      "privacy-statement": "#",
-      "data-policy": "#",
-      "contact": "#"
+      "facility": "https://www.isis.stfc.ac.uk/Pages/home.aspx",
+      "privacy-statement": "/cookies",
```
Comment (Pasarus) [inline container/default.json:132]: Does this exist right now? if so this is fine

**Review by Pasarus — CHANGES_REQUESTED** [review]
Just a quick change and question

File: container/default.json (hunk 2 above)
Comment (Dagonite) [reply container/default.json:132]: The cookies page? Yeah though have just kept as it is from SciGateway

**Review by Pasarus — APPROVED** [review]

**Pull request merged**